### DISCUSSION TITLE
💯稳定的接口分享💖

### DISCUSSION THREAD

**Discussion by Guovin** [Guovin/iptv-api#772] [Ideas]
分享您认为稳定有效的频道接口或订阅源，我将验证后添加，大家一起实现更好的观看体验~

**Comment by xxl6097** [comment]
搜源绝招

http://so.owo.vin/要搜的名字

例如：http://so.owo.vin/卫视
例如2: http://so.owo.vin/cctv


真心希望在接口里面加上这个！！！

**Reply by jasonlianrulin** [reply]
此方法已失效，请问是否还有新的方法

**Reply by opeopeopeopeopeopeopeopeopeopeopeopeope** [reply]
请看下面我打的

**Comment by opeopeopeopeopeopeopeopeopeopeopeopeope** [comment]
> 分享您认为稳定有效的频道接口或订阅源，我将验证后添加，大家一起实现更好的观看体验~

此前就有myIPTV了
项目地址:https://github.com/suxuang/myIPTV
🌐订阅链接-IPV6专用：https://gh.tryxd.cn/raw.githubusercontent.com/suxuang/myIPTV/main/ipv6.m3u
🌐订阅链接-IPV4专用：https://gh.tryxd.cn/raw.githubusercontent.com/suxuang/myIPTV/main/ipv4.m3u

**Reply by Guovin** [reply]
该数据源已经在默认订阅了

**Reply by opeopeopeopeopeopeopeopeopeopeopeopeope** [reply]
半失效，半没有失效
https://gh-proxy.com/github.com/wcb1969/iptv/raw/main/%E5%9F%8E%E5%BB%BA%E5%AD%A6%E9%99%A2TV.txt
